code/web/src/modules/user/StyleSurvey.js

```diff
@@ -14,25 +15,73 @@ import { white, grey, grey2, grey3 } from "../../ui/common/colors";
+    punk: "this thing",
+    artsy:
+      "You’re way of dressing is innovative and individualistic and you aren’t overly influenced by current trends or traditional rules. You use your clothing choices to reflect your personality, and put together unique and interesting outfits with items purchased from varied sources. Different colours, textures and prints fill your wardrobe and your jewellery collection is bold while your footwear and accessories are usually statement making. ",
+  };
+
   return (
     <section>
       <Helmet>
         <title>My Style Preferences</title>
       </Helmet>
-      <Grid style={{ backgroundColor: grey }}>
-        <GridCell style={{ padding: "2em", textAlign: "center" }}>
-          <H3 font="secondary">My Style Preferences</H3>
-
-          <p style={{ marginTop: "1em", color: grey2 }}>
-            Please complete the survey below.
-          </p>
-        </GridCell>
-      </Grid>
-      <GridCell style={{ textAlign: "center", backgroundColor: grey }}>
-        {/* <p style={{ marginBottom: '1em', color: grey2 }}>Like what you see?</p> */}
-        <StyleSurveyBox />
-      </GridCell>
+      {!props.user.stylePreference ? (
+        <div>
+          <Grid style={{ backgroundColor: grey }}>
+            <GridCell style={{ padding: "2em", textAlign: "center" }}>
+              <H3 font="secondary">My Style Preferences</H3>
+
+              <p style={{ marginTop: "1em", color: grey2 }}>
+                Please complete the survey below.
+              </p>
+            </GridCell>
+          </Grid>
+          <GridCell style={{ textAlign: "center", backgroundColor: grey }}>
+            {/* <p style={{ marginBottom: '1em', color: grey2 }}>Like what you see?</p> */}
+            <StyleSurveyBox />
+          </GridCell>
+        </div>
+      ) : (
+        <div>
+          <Grid style={{ backgroundColor: grey }}>
+            <GridCell style={{ padding: "2em", textAlign: "center" }}>
+              <H3 font="secondary">My Style Preferences</H3>
+            </GridCell>
+            <GridCell style={{ textAlign: "center", backgroundColor: grey }}>
+              <section
+                className="style-survey-box"
+                style={{ backgroundColor: "#f9f9f9" }}
+              >
+                <h3 style={{ marginTop: "2em" }}>
+                  Your style is:
+                  <br></br>~ {props.user.stylePreference} ~
+                </h3>
+                <br></br>
+                <h3 style={{ marginTop: "1em" }}>
+                  <img
+                    style={{
+                      height: "288px",
+                      width: "231px",
+                      objectFit: "cover",
+                      margin: "1%",
+                    }}
+                    src={`${APP_URL}/images/stylesurvey/${props.user.stylePreference}.jpg`}
+                  />
+                  {styleDescriptions[props.user.stylePreference]}
+                </h3>
```

Comment:
Specific interpulation for the user's stylePreference so that the results page is dynamic